osutil/disks/disks.go

```diff
@@ -147,6 +147,18 @@ type Partition struct {
 	KernelDevicePath string
 	// KernelDeviceNode is the kernel device node in /dev.
 	KernelDeviceNode string
+	// PartitionType is the type of structure, for example 0x0C.
```

Comment:
Should probably mention that it's lowercase/uppercase to avoid confusion
```
.. or <example gpt-type-uuid>.
```

Reply:
Done